tests/unit/src/utils/test_utils.py

```diff
@@ -13,5 +13,15 @@ async def stack(starknet):
 
 @pytest.mark.asyncio
 class TestStack:
-    async def test__bytes_i_to_uint256(self, stack):
-        await stack.test__bytes_i_to_uint256().call()
+    async def test__bytes_i_to_uint256(self, helpers):
+        await helpers.test__bytes_i_to_uint256().call()
+
+
+@pytest.mark.asyncio
+@pytest.mark.Utils
+class TestHelpers:
```

Comment:
looks like you can just remove this wrong `TestStack` and put everything under `TestHelpers` right?